### ISSUE TITLE
Training code for dlib face recognition resnet model

### ISSUE THREAD

**Issue by chensteven** [ageitgey/face_recognition#565]
Is there a training code or example available for the model dlib_face_recognition_resnet_model_v1 from https://github.com/davisking/dlib-models?

Also curious if there are any document on what the ResNet architecture look like and etc.

Thanks!

**Comment by ageitgey** [comment]
There are links to the network architecture and training code here: https://github.com/ageitgey/face_recognition/wiki/Face-Recognition-Accuracy-Problems#question-can-i-re-train-the-face-encoding-model-to-make-it-more-accurate-for-my-images

**Comment by fatalfeel** [comment]
pytorch main point is TripletLoss
https://github.com/tbmoon/facenet
https://arxiv.org/pdf/1503.03832.pdf theory
